src/main/resources/assets/computercraft/lua/rom/programs/fun/advanced/paint.lua

```diff
@@ -261,27 +261,58 @@ local function accessMenu()
     local selection = 1
 
     term.setBackgroundColour(colours.black)
+
+    local function selectOption( option )
+        if option == "Save" then
+            if bReadOnly then
+                fMessage = "Access denied"
+                return false
+            end
+            local success, err = save(sPath)
+            if success then
+                fMessage = "Saved to " .. sPath
+            else
+                if err then
+                    fMessage = "Error saving to " .. err
+                else
+                    fMessage = "Error saving to " .. sPath
+                end
+            end
+            return false
+        elseif option == "Exit" then
+            return true
+        end
+    end
+
     while true do
         -- Draw the menu
         term.setCursorPos(1, h)
         term.clearLine()
         term.setTextColour(colours.white)
+        tMenuPos = {}
+        local nStartPos
+        local nEndPos
         for k, v in pairs(mChoices) do
             if selection == k then
                 term.setTextColour(colours.yellow)
                 local ox = term.getCursorPos()
                 term.write("[" .. string.rep(" ", #v) .. "]")
                 term.setCursorPos(ox + 1, h)
+                nStartPos = ox + 1
                 term.setTextColour(colours.white)
                 term.write(v)
-                term.setCursorPos(term.getCursorPos() + 1, h)
+                nEndPos = term.getCursorPos()
```

Comment:
Is this missing a `+ 1`? We currently have this, which is a little broken:

![Slightly broken menu](https://user-images.githubusercontent.com/4346137/79954783-ba4efc00-8475-11ea-8490-8921288aa553.png)

That said, do we need to keep track of the positions? You don't appear to within edit, which does end up being a little shorter/cleaner.